### PR TITLE
Solution description section

### PR DESCRIPTION
**Pull request by alibombaye**
Updated marketing pages to display the description section.

### CODE REVIEW

File: app/helpers/determineStatusForSection.test.js [hunk 1]
```diff
@@ -1,69 +1,303 @@
...
+          },
+        ],
+      };
+
+      const status = determineStatusForSection(sectionManifest, sectionData);
+
+      expect(status).toEqual('INCOMPLETE');
+    });
+
+    it('should return COMPLETE if just one optional question of type bulletpoint-list and data has been provided', () => {
```
Comment (AndyBerryBJSS) [inline app/helpers/determineStatusForSection.test.js]: Not a bulletpoint list

File: app/helpers/validateSectionData.js [hunk 2]
```diff
@@ -24,14 +24,22 @@ export const validateSectionData = (sectionManifest, sectionData) => {
...
+          sectionData[sectionQuestion.id].map((sectionDataField, sectionDataFieldId) => {
+            if (validationRules[saveValidation.type].rule(sectionDataField, saveValidation)) {
+              const error = createErrorForField(
+                sectionQuestion.id, sectionDataFieldId, saveValidation.message,
+              );
+              validationErrors.push(error);
+            }
+          });
+        } else if (validationRules[saveValidation.type]
+          .rule(sectionData[sectionQuestion.id], saveValidation)) {
```
Comment (AndyBerryBJSS) [inline app/helpers/validateSectionData.js]: Null reference if no validation rule exists for the type

File: app/helpers/determineStatusForSection.test.js (hunk 1 above)
Comment (alibombaye) [reply app/helpers/determineStatusForSection.test.js]: updated

File: app/helpers/validateSectionData.js (hunk 2 above)
Comment (alibombaye) [reply app/helpers/validateSectionData.js]: added a guard to check there is a validation rule before executing the rule and added a test to check this works as expected

**Pull request merged**